integration/presto/pom.xml

```diff
@@ -431,14 +435,27 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-sql_2.11</artifactId>
+      <artifactId>spark-network-common_2.11</artifactId>
```

Comment:
Even though we remove all spark dependency in presto integration module, why here still need spark dependency in pom.xml of presto?

Reply:
For Running the Integration Tests as when we create a CarbonStore we need the Spark Dependency, so what we have done is that we have kept the scope of those dependency as Test , no spark Jar will be part of the final Jar that we copy to presto plugin

Reply:
ok, is it only for test cases , right ?

Reply:
Yes it is used only in Tests, we have defined the scope as test for all these dependencies